code/game/gamemodes/game_mode.dm

```diff
@@ -97,16 +99,16 @@
 ///convert_roundtype()
 ///Allows rounds to basically be "rerolled" should the initial premise fall through
 /datum/game_mode/proc/convert_roundtype()
-	var/living_crew = 0
+	var/list/living_crew = list()

```

Comment:
My only guess at this point is that  Byond has some sort of special handling for nonsensical "in" calls and just replaces it with "in world" at runtime without telling anyone.